.ci_support/osx_64_.yaml

```
@@ -13,21 +13,19 @@ curl:
 fftw:
 - '3'
 gdal:
-- '3.0'
+- '3.1'
 hdf5:
 - 1.10.5
```

```suggestion
hdf5:
- 1.10.6
```

Can we try bumping [hdf5](https://anaconda.org/conda-forge/hdf5/files) to 1.10.6?

```
Package hdf5 conflicts for:
gdal=3.1 -> hdf5[version='>=1.10.6,<1.10.7.0a0']
hdf5=1.10.5
libnetcdf=4.7.4 -> hdf5[version='*|*|*|>=1.10.6,<1.10.7.0a0|>=1.10.5,<1.10.6.0a0|>=1.10.6,<1.10.7.0a0|>=1.10.6,<1.10.7.0a0|>=1.10.5,<1.10.6.0a0|>=1.10.5,<1.10.6.0a0',build='mpi_openmpi_*|mpi_mpich_*|mpi_openmpi_*|mpi_openmpi_*|mpi_mpich_*|mpi_mpich_*|nompi_*']
```

(Ideally, we should wait for the hdf5 migration at https://conda-forge.org/status/#current_migrations, but it might not come).

---

I think these files are populated from `conda-forge-pinnings` to ensure compatibility across the ecosystem. So we might just have to wait for other packages to rebuild.

---

Ah ok, just that `gmt` appears to have a package conflict with `cartopy` as reported in https://forum.generic-mapping-tools.org/t/can-not-install-both-gmt-and-cartopy-because-of-conda-dependency-conflicts/963, and this PR *might* help to resolve it. I don't see `gmt` listed under https://conda-forge.org/status/#hdf51106 though, is there a way to manually add `gmt` to the list somehow?

---

That’s the point at which we call on the conda-forge elders for guidance (e.g. @ocefpaf)

---

According to the migration status gmt got an hdf5 PR but that did not happen :-/

See https://conda-forge.org/status/#hdf51106